constants.js

```diff
@@ -98,7 +99,8 @@ const checksWhitelist = {
     [periodicCheckEvent]: [periodicCheck],
     [pullRequestReviewEvent]: [pullRequestReviewCheck],
     [checkCompletedEvent]: [ciFailureCheck],
-    [issueCommentCreatedEvent]: [respondToReviewCheck]
+    [issueCommentCreatedEvent]: [respondToReviewCheck],
+    [ensureNewIssuesHaveProjects]: [ensureNewIssuesHaveProjects]
```

Comment:
I don't understand this, I thought that in this object we had a mapping between events and checks, why you map `ensureNewIssuesHaveProjects` to `ensureNewIssuesHaveProjects`?

Reply:
Actually in this I took names of both same, But will change it in the latest commit. Thanks for telling.